docker-compose.build.yaml

```diff
@@ -29,6 +29,13 @@ services:
       context: airbyte-workers
       labels:
         io.airbyte.git-revision: ${GIT_REVISION}
+  connector-builder:
```

Comment:
this is only for the local build - we also have to update https://github.com/airbytehq/airbyte/blob/master/tools/bin/publish_docker.sh#L5 to publish the image on releases

Reply:
done https://github.com/airbytehq/airbyte/pull/18410/commits/98c4a4141ed7f1113474bea7ae2a7147480c5c08